LoRaEngine/LoraKeysManagerFacade/LoraKeysManagerFacadeConstants.cs

```diff
@@ -11,6 +11,7 @@ internal static class LoraKeysManagerFacadeConstants
         internal const string TwinProperty_DevAddr = "DevAddr";
         internal const string TwinProperty_NwkSKey = "NwkSKey";
         internal const string NetworkServerModuleId = "LoRaWanNetworkSrvModule";
+        internal const string ClearCacheMethodName = "clearcache";
```

Comment:
Instead of introducing this, should we consider sharing it between the LNS and the Facade? Otherwise, we should have a test that breaks if we change the value of this constant.

Reply:
Unfortunately the constants are already spread between the two projects. I would keep the refactoring out of this PR